lib/web/ui/notification.go

```diff
@@ -36,7 +36,7 @@ type Notification struct {
 
 // MakeNotification creates a notification object for the WebUI.
 func MakeNotification(notification *notificationsv1.Notification) Notification {
-	labels := makeLabels(notification.Metadata.Labels)
+	labels := makeLabelsIncludingInternal(notification.Metadata.Labels)
```

Comment:
This is because for notifications, the notification text content and a few other user-facing values are stored in these labels, and they're required when building the response to send to the UI so it knows what to display